content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
 PriorityClass also has two optional fields: `globalDefault` and `description`.
 The `globalDefault` field indicates that the value of this PriorityClass should
 be used for Pods without a `priorityClassName`. Only one PriorityClass with
-`globalDefault`  set to true can exist in the system. If there is no PriorityClass
+`globalDefault` set to true should exist in the system. If there is more than one
+PriorityClass with `globalDefault` set to true, the value of the smallest
+of those PriorityClasses is used as the default priority. If there is no PriorityClass
 with `globalDefault` set, the priority of Pods with no `priorityClassName` is zero.
 
 The `description` field is an arbitrary string. It is meant to tell users of
 the cluster when they should use this PriorityClass.
 
+Kubernetes comes with two predefined PriorityClasses: `system-node-critical`, and
+`system-cluster-critical`. The former has the highest priority and is used for critical
+system Pods that must run on all or most of the nodes and should never
```

Comment:
nits: it'd like to say " system-node-critical ... critical system Pods that much run specific node and should never ...", as priority is bound to pod; "all or most of nodes" seems related to DaemonSet or Deployment, although that maybe the case :)